### ISSUE TITLE
FM: search button disappears in hamburger menu

### ISSUE THREAD

**Issue by luizaogs** [arch-kiosk/arch-kiosk-office#1758]
I will try to reconstruct this but opening the ticket now before I forget. I was testing the stuff in #1708 and moving around different loci and their interpretation and description tabs, and I opened the hamburger menu again and suddenly the search button was gone (photographic evidence below). I then went back to the start page and went back to the same unit (LA) and it had reappeared.

![IMG_0302](https://user-images.githubusercontent.com/67970402/180095456-ba29f831-ae7c-48e6-997a-ec5281173de4.PNG)

**Comment by luizaogs** [comment]
Ok yeah, so reconstruction of steps: 

Start on start page, go into unit LA (though it would happen with any unit, I assume).
In the excavating start layout, open the hamburger menu and search for "book"
Navigate to LA-005, surface collection (or, I assume, any other record)
Next return to the excavating layout by clicking on the back arrow, you'll end up in the context list for unit LA
Click on hamburger menu -- search button is gone
Go back to start page, navigate to unit LA
Click on the list of contexts for the unit --> the search button is back in the hamburger menu

**Comment by urapadmin** [comment]
What project is this? Is it possible that you start at excavating - simple and return to excavating (not simple)?

**Comment by luizaogs** [comment]
PVD. And I don't see how that would happen between the steps I outline above? I never even leave excavating, it just disappears while I'm still in there.

**Comment by luizaogs** [comment]
Oh sure, I always just think of the locus layout as part of excavating so it didn’t occur to me that a change might have occurred when I go to that layout.

**Comment by urapadmin** [comment]
i‘ll look into it. But first: Bed. Goodnight.

**Comment by luizaogs** [comment]
😴

**Comment by lbestock** [comment]
I noticed (after much similar confusion yesterday) that in PASU I did not have it unless I was actually in a locus. PASU is in simple excavating. So I could be in the locus list and not see it, and then click on a particular locus and see it. My assumption was that it just hadn't been put anywhere else yet. But I could not shake the feeling that, as Luiza notes, I had seen it originally present in the hamburger menu from when I first clicked on an area, before I did anything with loci. I then logged out and closed and reopened several times without being able to see the search available at the unit level, however, so I discount my uneasiness. 

And then it seemed to get weirder - I prepared and downloaded again the exact same database. And then I COULD see the search at the unit level and it does not disappear.

I tried all sort of things to maybe make it disappear how it had previously for me, including adding a new unit, a new locus, a new CM. Nothing made that magnifying glass disappear as it had previously.

I have both of these files still on my computer. The disappearing act is happening in one that was 13.5 and it is not in the one that is 13.5.1 and might that be the thing and nothing to worry about? But I still can't shake the feeling that I SAW it in 13.5 in a place that I later couldn't find it at all.
